api/tests/unit/features/workflows/core/test_unit_workflows_models.py

```diff
@@ -897,3 +977,31 @@ def _create_segment(percentage_value: int) -> Segment:
     )
     assert len(after_cr_1_flags) == 1
     assert after_cr_1_flags[0].feature_segment.segment == twenty_percent_segment
+
+
+def test_approval_via_project(project_change_request: ChangeRequest) -> None:
+    # Given - The project change request fixture
+    assert project_change_request.environment is None
+    assert project_change_request.project.minimum_change_request_approvals is None
+
+    # When
+    is_approved = project_change_request.is_approved()
+
+    # Then
+    assert is_approved is True
+
+
+def test_url_via_project(project_change_request: ChangeRequest) -> None:
+    # Given
+    assert project_change_request.environment is None
+
+    # When
+    url = project_change_request.url
+
+    # Then
+    project_id = project_change_request.project.id
```

Comment:
... feel bad nitpicking after all the other comments, but this is a particularly bad habit to get into with django, so I'm calling it out here, even though it's just test code. 

```suggestion
    project_id = project_change_request.project_id
```

Reply:
No problem, I've updated the code to match